tests/functional/spec/section_summary.spec.js

```diff
@@ -31,11 +32,11 @@ describe('Section Summary', function() {
     it('When I continue on the section summary page, Then I should be taken to the next section', function() {
       return browser
         .click(PropertyDetailsSummaryPage.submit())
-        .getUrl().should.eventually.contain(FinalSummaryPage.pageName);
+        .getUrl().should.eventually.contain(HouseType.pageName);
```

Comment:
Why has this route changed?

Reply:
The section summary schema had a section with two groups that could be skipped, resulting in an empty section, which doesn't work. The skip conditions were removed on the understanding this is not something we should be doing - skip conditions need to be added to sections to effectively fix this.